lib/cylc/remote.py

```diff
@@ -252,7 +252,7 @@ def remrun(dry_run=False, forward_x11=False, abort_if=None,
 class RemoteRunner(object):
     """Run current command on a remote host.
 
-    If owner or host differ from username and localhost, strip the
+    If user/owner or host differ from username and localhost, strip the
```

Comment:
I think "owner" and "user" were originally intended to distinguish the user that the suite daemon is running as (the suite "owner") from the user running a client command to connect to that suite daemon.  But maybe we lost site of that and haven't used the terms consistently since...

Reply:
I perhaps should have outlined that I changed it since the original lines 277-278 then become consistent with the form of the following two statements (i.e. for the other options taking option arguments) so that original 277-282 could all be consolidated, as applied. But if you would prefer me to leave the attribute name as is, I am happy to change it back. Or simply change ``user/owner`` to ``user`` for precision in this docstring?

Reply:
Fair enough, I will revert this in my branch.

Reply:
OK. That's a bit annoying. I guess you don't want to have to map between `user` and `owner` in the `setattr` method.

Reply:
Maybe leave the rename that you have done here - as long as it is safe and self-contained - and raise a follow-on issue to fix the rest of the consistency issues between *user* and *owner*.

Reply:
Exactly, the rename was to simplify the code. It is all self-contained here, yes, so I will go with your suggestion @matthewrmshin if @hjoliver is happy with that approach?